interface/ceed-elemrestriction.c

```diff
@@ -357,6 +357,78 @@ int CeedElemRestrictionCreate(Ceed ceed, CeedInt num_elem, CeedInt elem_size,
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Create a CeedElemRestriction with orientation sign
+
+  @param ceed         A Ceed object where the CeedElemRestriction will be created
+  @param num_elem     Number of elements described in the @a offsets array
+  @param elem_size    Size (number of "nodes") per element
+  @param num_comp     Number of field components per interpolation node
+                        (1 for scalar fields)
+  @param comp_stride  Stride between components for the same L-vector "node".
+                        Data for node i, component j, element k can be found in
+                        the L-vector at index
+                        offsets[i + k*elem_size] + j*comp_stride.
+  @param l_size       The size of the L-vector. This vector may be larger than
+                        the elements and fields given by this restriction.
+  @param mem_type     Memory type of the @a offsets array, see CeedMemType
+  @param copy_mode    Copy mode for the @a offsets array, see CeedCopyMode
+  @param offsets      Array of shape [@a num_elem, @a elem_size]. Row i holds the
+                        ordered list of the offsets (into the input CeedVector)
+                        for the unknowns corresponding to element i, where
+                        0 <= i < @a num_elem. All offsets must be in the range
+                        [0, @a l_size - 1].
+  @param orient       Array of shape [@a num_elem, @a elem_size] with bool false
+                        for positively oriented and true to flip the orientation.
+  @param[out] rstr    Address of the variable where the newly created
+                        CeedElemRestriction will be stored
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref User
+**/
+int CeedElemRestrictionCreateOriented(Ceed ceed, CeedInt num_elem,
+                                      CeedInt elem_size, CeedInt num_comp,
+                                      CeedInt comp_stride, CeedInt l_size,
+                                      CeedMemType mem_type, CeedCopyMode copy_mode,
+                                      const CeedInt *offsets, const bool *orient,
+                                      CeedElemRestriction *rstr) {
+  int ierr;
+
+  if (!ceed->ElemRestrictionCreate) {
```

Comment:
Woops, this should probably be 
```suggestion
  if (!ceed->ElemRestrictionCreateOriented) {
```

That should fix the CI issue (probably want to rebase off main though because CI has changed)

Reply:
Ah, thanks!